src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Startup.cs

```diff
@@ -159,10 +159,10 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
             app.UseRouting(routes =>
             {
-                routes.MapApplication();
                 routes.MapControllerRoute(
```

Comment:
Should this change to MapDefaultControllerRoute?

Reply:
I looked at the pre-endpoint templates and they included the full route like this. Never mind!

Reply:
Yep!